torch/csrc/autograd/python_engine.cpp

```diff
@@ -191,7 +193,9 @@ PyObject *THPEngine_run_backward(THPEngine *self, PyObject *args, PyObject *kwar
     }
     // Disable execution for all unneeded functions
     if (only_inputs) {
-      compute_partial_exec_callbacks(roots, ctx, callbacks);
+      if (!compute_partial_exec_callbacks(roots, ctx, callbacks)) {
```

Comment:
Can't we just wrap the engine's function in `HANDLE_TH_ERRORS`? It should be there anyway and it's a clearer way of handling errors